### PR TITLE
add docs on supported regions

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by komarkovich — APPROVED** [review]
LGTM

File: content/en/references/regions-coverage.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+description: >
+    This page lists the AWS Region Coverage for LocalStack's emulation of AWS services.
+cascade:
+  type: docs
+---
+
+## Introduction
+
+LocalStack ships with multi-region support, enabling users to emulate different AWS regions and namespace resources based on the region.
+With IAM Policy Enforcement, you can ensure that access control and permissions are applied accurately across supported regions.
```
Comment (dfangl) [inline content/en/references/regions-coverage.md:14]: I am not sure about that statement, it does not seem to quite fit here? I mean yes, you can use it with IAM enforcement, but it should not make a difference.

**Review by dfangl — APPROVED** [review]
LGTM! Just one comment about mentioning IAM enforcement.

**Review by dfangl — COMMENTED** [review]
I just double checked, the two region mentioned as not supported work fine for me (at least for sqs), could you verify that?

**Pull request merged**